src/main/java/org/mule/extension/sftp/internal/connection/SftpClient.java

```diff
@@ -362,6 +362,20 @@ public void write(String path, InputStream stream, FileWriteMode mode) throws Ex
     sftp.put(stream, path, toInt(mode));
   }
 
+  /**
+   * Returns the home directory of the sftp server
+   *
+   * @return a {@link String}
+   */
+  public String getHome() {
+    try {
+      return sftp.getHome();
+    } catch (SftpException e) {
+      LOGGER.warn("Home directory could not be acquired", e);
+      return "";
```

Comment:
What happens if this method returns an empty string? Is it the same situation that we had before?

Reply:
Yes it would be the same situation and a / would be added at beginning.